opencga-app/app/scripts/azure/arm/daemonvm/azuredeploy.json

```diff
@@ -120,7 +120,7 @@
         "subnetRef": "[concat(variables('vnetId'), '/subnets/', parameters('subnetName'))]",
         "solrHostsCSV": "[concat('\"', replace(replace(replace(replace(string(parameters('solrHosts')), ' ', ''), '[{\"host\":\"', ''), '\"},{\"host\":\"', ','), '\"}]', ''),'\"')]",
         "mongoDbHostsCSV": "[concat('\"', replace(replace(replace(replace(string(parameters('mongoDbHosts')), ' ', ''), '[{\"fqdn\":\"', ''), '\"},{\"fqdn\":\"', ','), '\"}]', ''),'\"')]",
-        "webserverHost": "[concat('http://', parameters('webserverNamingPrefix'), '.', resourceGroup().location,'.cloudapp.azure.com')]",
+        "webserverHost": "[parameters('opencgaFQDN')]",

```

Comment:
I think this is expecting the format to be `http://`. Does the FQDN include the protocol?

Reply:
Doh. Good spot. @jjcollinge this may have broken stuff in the deployment you have access to. Will add protocol now.

Reply:
Also its not on 8080... via the app gateway. @jjcollinge should I change the 8080 references?

Reply:
Done